app/server/functions/src/main/kotlin/com/supercilex/robotscouter/server/functions/Templates.kt

```diff
@@ -50,32 +50,38 @@ fun matchTemplateMetrics() = metrics {
     header("a", "Scout info")
     text("b", "Name")
 
-    header("c", "Auto")
-    checkbox("d", "Crossed baseline")
-    selector("e", "Put cube in Switch") {
-        +Item("a", "Didn't attempt")
-        +Item("b", "Attempted")
-        +Item("c", "Successful")
-        +Item("d", "Unknown", true)
+    header("c", "Sandstorm")
+    selector("d", "Starting location") {
+        +Item("a", "HAB Level 2")
+        +Item("b", "HAB Level 1")
+        +Item("c", "Unknown", true)
     }
-    selector("f", "Put cube in Scale") {
-        +Item("a", "Didn't attempt")
-        +Item("b", "Attempted")
-        +Item("c", "Successful")
-        +Item("d", "Unknown", true)
+    checkbox("e", "Successfully crossed HAB line")
+    counter("f", "Panels on Cargo Ship")
+    counter("g", "Panels on low Rocket Hatches")
+    counter("h", "Panels on middle/high Rocket Hatches")
+    counter("i", "Cargo in Cargo Ship")
+    counter("j", "Cargo in low Rocket Bays")
+    counter("k", "Cargo in middle/high Rocket Bays")
+
+    header("l", "Teleop")
+    counter("m", "Panels on Cargo Ship")
+    counter("n", "Panels on low Rocket Hatches")
+    counter("o", "Panels on middle/high Rocket Hatches")
+    counter("p", "Cargo in Cargo Ship")
+    counter("q", "Cargo in low Rocket Bays")
+    counter("r", "Cargo in middle/high Rocket Bays")
+    stopwatch("s", "Cycle time")
```

Comment:
Should this be more specific?

Reply:
The cycle time?

Reply:
If so, maybe one timer for hatch panel cycles and another for cargo?

Reply:
Yeah, maybe. Or do you think robots will alternate and it'll be easier to scout a general sense of cycle time? (In which case the current metric is fine as is.)

Reply:
Having two metrics for the cycle timer might make it easier to keep track of which cycles are which. I'd say have two of 'em unless there's a significant drawback I'm not thinking of

Reply:
Nah, sounds good. 👍